includes/wikia/tests/UserTest.php

```diff
@@ -199,4 +213,47 @@ public function testGetUsernameShouldReturnDefaultValueIfUserIsNotFound() {
 		$this->mockStaticMethod( 'User', 'whoIs', false );
 		$this->assertEquals( 'someName', User::getUsername( 123, 'someName' ) );
 	}
+
+	public function testNewFromTokenAuthorizationGranted() {
+		$webRequestMock = $this->getMock( 'WebRequest', [ 'getCookie', 'getHeader' ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "getCookie" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)